ci-operator/jobs/ibmz-eco/isv-operator/ibmz-eco-isv-operator-master-periodics.yaml

```diff
@@ -0,0 +1,85 @@
+periodics:
+- agent: kubernetes
+  cluster: build01
+  decorate: true
+  interval: 2h
+  labels:
+    ci-operator.openshift.io/prowgen-controlled: "true"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
```

Comment:
This does not work the way you are trying to set it up. In OpenShift CI, Prow jobs that use multi-stage steps are generated from ci-operator configuration which describes how a repo is tested. Please see https://docs.ci.openshift.org/docs/architecture/ci-operator/ and https://docs.ci.openshift.org/docs/architecture/step-registry/ for more information.

Adding jobs with inline `CONFIG_SPEC` is not the way to do it.